nlp/drivers/nlp_auglag.py

```diff
@@ -51,6 +52,7 @@ def auglag_stats(auglag):
     model = PySparseAmplModel(problem)
     model.compute_scaling_obj()
     auglag = Auglag(model, TRON, maxiter=100)
+    pdb.set_trace()
```

Comment:
I know, one or two slipped through the cracks. I believe I fixed it in a later commit.